### PR TITLE
SDESK-5479 Update: admin delete disabled set with no asset

### PR DESCRIPTION
**Pull request by dev-tsoni**

### CODE REVIEW

File: scripts/extensions/sams/src/components/sets/setListItem.tsx [hunk 1]
```diff
@@ -78,7 +78,7 @@ export class SetListItem extends React.PureComponent<IProps> {
                     </ListItemRow>
                 </ListItemColumn>
                 <ListItemActionMenu row={true}>
-                    {this.props.deleteSet == null ? null : (
+                    {this.props.deleteSet == null || set.state === SET_STATE.DISABLED && count ? null : (
```
Comment (MarkLark86) [inline scripts/extensions/sams/src/components/sets/setListItem.tsx:81]: I'm not sure checking the state here is required. If the `count > 0` then we should never allow deleting the Set regardless of it's state

File: scripts/extensions/sams/src/components/sets/setPreviewPanel.tsx [hunk 2]
```diff
@@ -59,7 +61,7 @@ export function getSetPreviewPanel(superdesk: ISuperdesk) {
                 ariaValue: 'close',
             }];
 
-            if (set.state === SET_STATE.DRAFT) {
+            if (set.state === SET_STATE.DRAFT || set.state === SET_STATE.DISABLED && !count) {
```
Comment (MarkLark86) [inline scripts/extensions/sams/src/components/sets/setPreviewPanel.tsx:64]: Same here

File: scripts/extensions/sams/src/components/sets/setListItem.tsx (hunk 1 above)
Comment (dev-tsoni) [reply scripts/extensions/sams/src/components/sets/setListItem.tsx]: Resolved

File: scripts/extensions/sams/src/components/sets/setPreviewPanel.tsx (hunk 2 above)
Comment (dev-tsoni) [reply scripts/extensions/sams/src/components/sets/setPreviewPanel.tsx]: Resolved

**Review by MarkLark86 — APPROVED** [review]

**Pull request merged**